apps/finance/app/src/components/LocalIdentityModal/LocalIdentityModalManager.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Subject } from 'rxjs'
+
+const updates$ = new Subject()
```

Comment:
Same thing than [in the `aragon/aragon` PR](https://github.com/aragon/aragon/pull/656#discussion_r268710971) about RxJS, but I think it is especially relevant for apps as they also serve as examples for app developers.

Reply:
I'll open an issue to keep track of this (and for all the apps too).